classes/class-cs-loader.php

```diff
@@ -303,12 +303,14 @@ public function cswp_save_form_data() {
 		} elseif ( 'apirate' === $_POST['cswp_form_select'] ) {
 
 			$data = '';
-			$data = add_query_arg(
-				array(
-					'app_id' => $api_key,
-					'base'   => $basecurency,
-				),
-				'https://openexchangerates.org/api/latest.json'
+			$data = esc_url(
```

Comment:
Use `esc_url_raw` OR `sanitize_url` instead of esc_url.

Reply:
esc_url_raw() used instead of esc_url.